MiniTwitch.Irc/Models/Privmsg.cs

```diff
@@ -225,6 +230,10 @@ internal Privmsg(ref IrcMessage message, IrcClient? source = null)
                             gigantified = true;
                             break;
 
+                        case 1939:
+                            highlighted = true;
```

Comment:
This assumes that the `msg-id` tag always has the value of `highlighted-message`.
Perhaps add an if check or a switch:
```cs
if (tagValue.Sum() == 1939)
{
	highlighted = true;
}
```